src/views/studio/VideoWorkspace/VideoWorkspace.style.ts

```diff
@@ -2,8 +2,33 @@ import styled from '@emotion/styled'
 import { animated } from 'react-spring'
 
 import { oldColors, zIndex } from '@/styles'
+import { cVar } from '@/styles'
+import { VIDEO_WORKSPACE_TABS_BAR_HEIGHT } from '@/views/studio/VideoWorkspace/VideoWorkspaceTabsBar'
 
-export const Container = styled(animated.div)`
+export const DrawerOverlay = styled(animated.div)`
```

Comment:
Does tis still need to be `animated.div`? We're no longer using `react-spring`, right?